p2panda-auth/src/group/mod.rs

```diff
@@ -65,30 +59,31 @@ where
 
 /// Decentralised Group Management (DGM).
```

Comment:
DGM in the DCGKA paper stands for "Decentralised Group Membership". Not sure if we want to be consistent here, up to you!